### PR TITLE
Set up application structure for Admin Client

### PR DESCRIPTION
**Pull request by johanperjulius1**
## This PR contains:
- Scaffolding react application 
- Changing App.js to App.jsx 
- Changing App to a class
- Cleaning.

## [PT](https://www.pivotaltracker.com/story/show/175139264)

### CODE REVIEW

File: README.old.md [hunk 1]
```diff
@@ -0,0 +1,2 @@
+# client_admin_good_morning_news
+Newsroom Challenge August 2020 Team 1
```
Comment (emtalen) [inline README.old.md]: you can remove this

File: public/index.html [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+      If you open it directly in the browser, you will see an empty page.
+
+      You can add webfonts, meta tags, or analytics to this file.
+      The build step will place the bundled scripts into the <body> tag.
+
+      To begin the development, run `npm start` or `yarn start`.
+      To create a production bundle, use `npm run build` or `yarn build`.
+    -->
+  </body>
+</html>
```
Comment (emtalen) [inline public/index.html:20]: clean up the comments in here please

File: public/robots.txt [hunk 3]
```diff
@@ -0,0 +1,3 @@
+# https://www.robotstxt.org/robotstxt.html
+User-agent: *
+Disallow:
```
Comment (emtalen) [inline public/robots.txt]: remove this file please

File: src/App.css [hunk 4]
```diff
@@ -0,0 +1,38 @@
...
+}
+
+@keyframes App-logo-spin {
+  from {
+    transform: rotate(0deg);
+  }
+  to {
+    transform: rotate(360deg);
+  }
+}
```
Comment (emtalen) [inline src/App.css]: remove this file please

File: src/App.test.js [hunk 5]
```diff
@@ -0,0 +1,9 @@
+import React from 'react';
+import { render } from '@testing-library/react';
+import App from './App';
+
+test('renders learn react link', () => {
+  const { getByText } = render(<App />);
+  const linkElement = getByText(/learn react/i);
+  expect(linkElement).toBeInTheDocument();
+});
```
Comment (emtalen) [inline src/App.test.js]: remove this file please

**Review by emtalen — CHANGES_REQUESTED** [review]
also remove the logo images in the public folder.

File: src/App.jsx [hunk 6]
```diff
@@ -0,0 +1,13 @@
+import React, { Component } from 'react'
+
+ class App extends Component {
```
Comment (emtalen) [inline src/App.jsx]: please change this to a functional component, we will move away from class components

File: src/App.css (hunk 4 above)
Comment (johanperjulius1) [reply src/App.css]: Should we remove the index.css as well?

Comment (emtalen) [reply src/App.css]: No you can keep that

Comment (emtalen) [reply src/App.css]: @johanperjulius1

**Review by emtalen — APPROVED** [review]

**Pull request merged**